app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Post;
+use Illuminate\Http\Request;
+
+class PostController extends Controller
+{
+    public function createpost(Request $request){
+        $incomingFields = $request->validate([
+            'body' => 'required',
+            'title' => 'required'
+        ]);
+        
+        $incomingFields['title'] = strip_tags($incomingFields['title']);
+        $incomingFields['body'] = strip_tags($incomingFields['body']);
```

Comment:
No need to use strip_tags.
As far as I know this is already handled by Laravel's Request class automatically.